src/views/catalog/wizard/tabs/disks/hooks/useEditDiskState.ts

```diff
@@ -34,11 +34,12 @@ type UseEditDiskStates = (
   initialDiskSourceState: DiskSourceState;
 };
 
-const getSourceFromDataVolue = (dataVolume: V1DataVolumeTemplateSpec): string => {
+const getSourceFromDataValue = (dataVolume: V1DataVolumeTemplateSpec): string => {
   if (dataVolume.spec?.source?.blank) return sourceTypes.BLANK;
   else if (dataVolume.spec?.source?.http) return sourceTypes.HTTP;
   else if (dataVolume.spec?.source?.pvc) return sourceTypes.CLONE_PVC;
   else if (dataVolume.spec?.source?.registry) return sourceTypes.REGISTRY;
+  else if (dataVolume.spec?.sourceRef) return sourceTypes.DATA_SOURCE;
```

Comment:
I would put it first because its the most common source of common-templates

Reply:
changed :)